intercom-provider/build.gradle

```diff
@@ -53,10 +53,10 @@ android {
 dependencies {
 	implementation project(path: rootProject.ext.analytics_kit)
 	api "org.jetbrains.kotlin:kotlin-stdlib:$rootProject.ext.kotlinVersion"
-	api "androidx.annotation:annotation:1.2.0"
-	compileOnly 'io.intercom.android:intercom-sdk-base:10.6.1'
+	api "androidx.annotation:annotation:$rootProject.ext.annotationsVersion"
+	compileOnly 'io.intercom.android:intercom-sdk-base:12.0.0'
```

Comment:
There is a newer version of the Intercom SDK available, but updating to the latest version will require updating more test code than I was willing to touch in this PR. Let's update this SDK version in a follow-up PR.